test/unit/packages/fs.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
+
+import isJsFile from '../../../src/packages/fs/utils/is-js-file';
+
+describe('Unit: class Fs ', () => {
```

Comment:
We should use `class fs` since it's always imported that way (even though classes are traditionally upper camel cased).